app/assets/javascripts/index/contextmenu.js

```diff
@@ -39,7 +37,7 @@ OSM.initializeContextMenu = function (map) {
     callback: function describeLocation(e) {
       const [lat, lon] = OSM.cropLocation(e.latlng, map.getZoom()).map(encodeURIComponent);
```

Comment:
`encodeURIComponent` here probably doesn't do anything

Reply:
Maybe, but it's not something new in this pull request?